packages/reactive-element/src/reactive-element.ts

```diff
@@ -48,8 +50,9 @@ let requestUpdateThenable: (name: string) => {
 
 let issueWarning: (code: string, warning: string) => void;
 
-const trustedTypes = (global as unknown as {trustedTypes?: {emptyScript: ''}})
-  .trustedTypes;
+const trustedTypes = (
+  globalThis as unknown as {trustedTypes?: {emptyScript: ''}}
```

Comment:
This file already has `const global = globalThis;` up top. Do we need this replaced?